docs/application/dotnet/guides/messaging/push.md

```diff
@@ -284,7 +284,8 @@ To register with the push server:
     ```
 
     > **Note**   
-	> The `PushServiceUnregister()` method is not used if the application is intended to receive push notifications continuously while it is installed on the device. When the application is uninstalled, the push service detects the event and deregisters the application automatically.
+	> The `PushServiceUnregister()` method is not used if the application is intended to receive push notifications continuously while it is installed on the device. When the application is uninstalled, the push service detects the event and deregisters the application automatically.  
```

Comment:
Please see
https://github.com/JINSEONGHUN/tizen-docs/blob/dfae7683b90ea8e04951cb0de53cc8b5ee3cea27/docs/application/dotnet/guides/messaging/push.md.

Note tag have to be indented. It means that the number of spaces in front of the sentences in Note have to be same.